src/components/list-lemplate/list-template.tsx

```diff
@@ -0,0 +1,52 @@
+import { Dispatch, FC, SetStateAction } from "react";
+import { Department, Skill } from "cv-graphql";
+import { Button, Input } from "antd";
+import { t } from "i18next";
+import { SearchOutlined } from "@ant-design/icons";
+import TableTemplate, { ColumnConfig } from "./table-template";
+import { Action } from "./actions-menu";
+
+interface PageTemplateProps<T> {
+    pageName: string;
+    onButtonClick: () => void;
+    menuProps: Action;
+    columnConfigs: ColumnConfig<T>[];
+    searchQuery: string;
+    displayData: T[];
+    loading: boolean;
+    setSearchQuery: Dispatch<SetStateAction<string>>;
+}
+
+const PageTemplate: FC<PageTemplateProps<Skill | Department>> = ({
+    pageName,
+    onButtonClick,
+    menuProps,
+    columnConfigs,
+    searchQuery,
+    displayData,
+    loading,
+    setSearchQuery,
+}) => {
```

Comment:
Please ensure consistent prop typing style
we use
const PageTemplate= ({ props }: PageTemplateProps) => {
instead of React.FC<>

Reply:
fixed